srml/session/src/lib.rs

```diff
@@ -134,6 +134,10 @@ pub trait ShouldEndSession<BlockNumber> {
 	fn should_end_session(now: BlockNumber) -> bool;
 }
 
+/// Ends the session after a fixed period of blocks.
+///
+/// The first session will have length of `Period + Offset`, and 
```

Comment:
Why make first session special? Can't we handle it in a way that it's always `Period`?

Reply:
Well, one session has to start at 0. All others are supposed to start on multiples of `Offset`. Then it's impossible to do as long as `Period % Offset != 0`.

The docs here are wrong also...

Reply:
Right, thanks for clarifications!